### ISSUE TITLE
ddnet game skin - hook overlaps with hammer, and shotgun shield also off-center

### ISSUE THREAD

**Issue by WaRNe0nx3** [ddnet/ddnet#8942]
Hook is off-center and overlaps with the hammer area, causing black pixels where they shouldn't be, also the shotgun shield was a little bit off-center.

![ddnet_hammer_pixels](https://github.com/user-attachments/assets/0f309988-853a-4301-a45b-535021a35102)
![ddnet_gameskin_bugs](https://github.com/user-attachments/assets/dacbdc51-8505-4a64-8ad1-ed495e6bebbf)

I tried to fix it.
I lifted the hook by 1 pixel, so it's now centered in it's designated area, and doen't overlap into the hammer area.
I moved the shotgun shield to the left by to pixels to make it less off-center.

![ddnet_gameskin_fixed](https://github.com/user-attachments/assets/dcb95ca0-5a93-4e13-ad73-03aa7619d356)

**Comment by ChillerDragon** [comment]
I wonder when this bug was introduced. Or was it always there?

We should wait for the ddnet svg remake https://github.com/ddnet/ddnet-data-svg/pull/32 and make sure it produces the correct output.

**Comment by WaRNe0nx3** [comment]
I would like to add that the hook tip in itself is not symmetrical, also in the vector graphic you linked.

I hope I may also add this here:
The hammer crosshair is perfectly symmetrical as a vector graphic, but is not symmetrical in the game.png, I would say. Same as in official teeworlds 0.7.
![hammer_crosshair](https://github.com/user-attachments/assets/91e89da8-d9ee-4d68-90c8-c314e2ca7689)

**Comment by WaRNe0nx3** [comment]
i didn't mean to close this

**Comment by Jupeyy** [comment]
Yeah svg fixes cursors and hook pos too. We could use that if somebody adds the weapon shields

**Comment by Robyt3** [comment]
See also more issues in #11029.

The game svg was updated in https://github.com/ddnet/ddnet-data-svg/pull/32, so should we simply update the png based on the svg or are there any other issues?
